releasenotes/notes/add-rds-autodiscovery-8c0a6b0f2fe9a8c8.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    Add support for Autodicovery for RDS Postgres and MySQL databases.
```

Comment:
```suggestion
    Add support for automatic detection of RDS Postgres and MySQL databases.
```

Reply:
We used the same `Autodiscovery` term for [Aurora](https://docs.datadoghq.com/database_monitoring/guide/aurora_autodiscovery/?tab=postgres), so aiming to keep it consistent with that.